test/form_builder/general_test.rb

```diff
@@ -139,7 +139,11 @@ def with_custom_form_for(object, *args, &block)
 
   test 'builder generates uuid fields for uuid columns' do
     with_form_for @user, :uuid
-    assert_select 'form input#user_uuid.string.uuid'
+    if defined? ActiveModel::Type
+      assert_select 'form input#user_uuid.string.string'
```

Comment:
In the stub `type_for_attribute` implementation we are returning the `:string` type for `:uuid` columns, so Simple Form will generate the class name as `string` - as the class name is derived from the `type.type` method - production apps probably won't be affected by this because the type object will likely be a [`ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Uuid`](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/postgresql/oid/uuid.rb) one.